code/drasil-example/Drasil/HGHC/HeatTransfer.hs

```diff
@@ -31,6 +34,10 @@ cladCond     = vc "cladCond"     (cnIES "clad conductivity") (lK `sub` lC) Real
 htTransCladCool_eq, htTransCladFuel_eq :: Expr
 htTransCladCool, htTransCladFuel :: QDefinition
 
+htTransCladCoolDD :: DataDefinition
+htTransCladCoolDD = mkDD htTransCladCool [] [] ""
```

Comment:
Why do these constructors have `[] [] ""`? Is that just because tiny is lacking a lot of information, or is it common for that to happen (and we should create a new smart constructor if that's the case)?

Reply:
The `[] [] "" `  is for the references in the source field and derivation, while the empty string at the end seems to be temporary. These fields are left empty because tiny is lacking information, the Data Definitions for tiny didn't have references or derivations so tiny is lacking information in that regard.